py/requirements.txt

```diff
@@ -0,0 +1,10 @@
+Jinja2==2.9.5
```

Comment:
Replace `==` with `>=` would be better. We don't need to specify an exact version of dependencies. Same as below.

Reply:
OK I will update it, but I am not sure if all the denpendencies are upwardly compatible.

Reply:
Yep, so we have to guarantee each items.